README.md

```diff
@@ -30,13 +30,13 @@ this package to execute our complex `dcp-client` library, which is written in JS
 - [done] JS functions coerce to Python function wrappers
 - [done] JS exceptions propagate to Python
 - [done] Implement `eval()` function in Python which accepts JS code and returns JS->Python coerced values
-- [underway] NodeJS+NPM-compatible CommonJS module system
+- [done] NodeJS+NPM-compatible CommonJS module system
 - [done] Python strings coerce to JS strings
 - [done] Python intrinsics coerce to JS intrinsics
 - Python dicts coerce to JS objects
-- Python `require` function, returns a coerced dict of module exports
+- [done] Python `require` function, returns a coerced dict of module exports
 - [done] Python functions coerce to JS function wrappers
-- CommonJS module system .py loader, loads Python modules for use by JS
+- [done] CommonJS module system .py loader, loads Python modules for use by JS
 - JS object->Python dict coercion supports inherited-property lookup (via __getattribute__?)
 - Python host environment supplies event loop, including EventEmitter, setTimeout, etc.
 - Python host environment supplies XMLHttpRequest (other project?)
```

Comment:
I'd like to move the roadmap to possibly https://github.com/orgs/Distributive-Network/projects/1 so that we don't have to update the README often.